test/Microsoft.ML.Tests/TrainerEstimators/MetalinearEstimators.cs

```diff
@@ -97,7 +97,7 @@ public void Pkpd()
         {
             var dataPath = GetDataPath(IrisDataPath);
 
-            using (var env = new TlcEnvironment())
+            using (var env = new ConsoleEnvironment())
```

Comment:
>env [](start = 23, length = 3)

you can use Env from base class, but it's unrelated to your PR

Reply:
Indeed. 

---
In reply to: [217868501](https://github.com/dotnet/machinelearning/pull/923#discussion_r217868501) [](ancestors = 217868501)

Reply:
I'll fix it. 

---
In reply to: [217869448](https://github.com/dotnet/machinelearning/pull/923#discussion_r217869448) [](ancestors = 217869448,217868501)